### PR TITLE
Doc updates to support new registration template for SSR

### PR DESCRIPTION
**Pull request by robertnicolo-okta**
## Publish date

Sometime in **July, 2022** pending a security review

## Description:
Doc updates to support new registration template for SSR

### Resolves:

* [OKTA-499822](https://oktainc.atlassian.net/browse/OKTA-499822)

### Deploy link (Updated 06/02)

https://629907bff431bf12b63d897c--reverent-murdock-829d24.netlify.app/docs/guides/authenticators-okta-email/aspnet/main/#know-which-use-cases-support-magic-link-and-otp

### CODE REVIEW

**Review by danielmaharry-okta — APPROVED** [review]
Looks good to me Rob!

File: packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/index.md [hunk 1]
```diff
@@ -89,12 +89,12 @@ Your app should always display an OTP input page to allow the user to use the OT
...
-| Email Challenge             | Sign-in with email - challenge                           | Email challenge   | [Magic link](#integrate-email-challenge-with-magic-links), OTP  | No
-| Forgot Password             | Self-service password recovery                           | Email challenge   | [Magic link](#integrate-email-challenge-with-magic-links), OTP  | Yes, add `otp` and `state` parameters. See <StackSnippet snippet="custompwdguide" inline />.
-| Email Factor Verification   | Self-service registration, Sign-in with email - enroll   | Email enrollment  | Magic link, [OTP](#integrate-email-enrollment-with-otp)              | No
-
+| Email template name    |   Use cases    |   Supported methods | Template customizations supported |
+| ----------------------------| ------------------|-------------|-------------------------|
+| Email Challenge             | Sign-in with email - challenge  | [Magic link](#integrate-email-challenge-with-magic-links), [OTP](#integrate-email-enrollment-with-otp)  | Yes, template supports the `otp` and `state` parameters. See the <StackSnippet snippet="custompwdguide" inline /> for more details.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/index.md:94]:
```suggestion
| Email Challenge             | Sign-in with email - challenge  | [Magic link](#integrate-email-challenge-with-magic-links), [OTP](#integrate-email-enrollment-with-otp)  | Yes, template supports the `otp` and `state` parameters. See <StackSnippet snippet="custompwdguide" inline />.
```

File: packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/index.md [hunk 2]
```diff
@@ -89,12 +89,12 @@ Your app should always display an OTP input page to allow the user to use the OT
...
+| Forgot Password             | Self-service password recovery  | [Magic link](#integrate-email-challenge-with-magic-links), [OTP](#integrate-email-enrollment-with-otp)  | Yes, template supports the `otp` and `state` parameters. See the <StackSnippet snippet="custompwdguide" inline /> for more details.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/index.md:95]:
```suggestion
| Forgot Password             | Self-service password recovery  | [Magic link](#integrate-email-challenge-with-magic-links), [OTP](#integrate-email-enrollment-with-otp)  | Yes, template supports the `otp` and `state` parameters. See <StackSnippet snippet="custompwdguide" inline />.
```

File: packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/index.md [hunk 3]
```diff
@@ -89,12 +89,12 @@ Your app should always display an OTP input page to allow the user to use the OT
...
+| Email Factor Verification   | Sign-in with email - enroll   | [Magic link](#integrate-email-challenge-with-magic-links), [OTP](#integrate-email-enrollment-with-otp) | Yes, template supports the `otp` and `state` parameters. See the <StackSnippet snippet="custompwdguide" inline /> for more details.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/index.md:96]:
```suggestion
| Email Factor Verification   | Sign-in with email - enroll   | [Magic link](#integrate-email-challenge-with-magic-links), [OTP](#integrate-email-enrollment-with-otp) | Yes, template supports the `otp` and `state` parameters. See <StackSnippet snippet="custompwdguide" inline />.
```

File: packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/index.md [hunk 4]
```diff
@@ -89,12 +89,12 @@ Your app should always display an OTP input page to allow the user to use the OT
...
+| Registration - Activation   | Self-service registration  | [Magic link](#integrate-email-challenge-with-magic-links), [OTP](#integrate-email-enrollment-with-otp)  | Yes, template supports the `otp` and `state` parameters. See the <StackSnippet snippet="custompwdguide" inline /> for more details.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/index.md:97]:
```suggestion
| Registration - Activation   | Self-service registration  | [Magic link](#integrate-email-challenge-with-magic-links), [OTP](#integrate-email-enrollment-with-otp)  | Yes, template supports the `otp` and `state` parameters. See <StackSnippet snippet="custompwdguide" inline />.
```

**Review by susanharper-okta — APPROVED** [review]
just a style nit. approving to not hold you up.

**Pull request closed without merging**